packages/async/async.109.19.00/opam

```diff
@@ -16,4 +16,7 @@ depends: [
   "async_unix" {= "109.19.00"}
   "ocamlbuild" {build}
 ]
-ocaml-version: [>= "4.00.0"]
+available: [ ocaml-version >= "4.00.0" & ocaml-version < "4.03" ]
+bug-reports: "https://github.com/janestreet/opam/issues"
+dev-repo: "https://github.com/janestreet/opam.git"
```

Comment:
The `bug-reports` and `dev-repo` fields all seem to be set to `https://github.com/janestreet/opam/issues` and `https://github.com/janestreet/opam.git`.